content/blog/2022-05-24-MLEM-release.md

```diff
@@ -0,0 +1,278 @@
+against some dataset directly in the command line, or serve the model with MLEM
+from your CLI.
+
+```python
+$ python
+>>> import mlem
+>>> model = mlem.api.load("mlem-model")
+>>> model.predict(test)
+[[0.4, 0.3, 0.3], [0.2, 0.5, 0.3]]
+```
+
+```shell
+$ mlem apply mlem-model test.csv
+[[0.4, 0.3, 0.3], [0.2, 0.5, 0.3]]
+```
+
+```shell
+$ mlem serve ml-model fastapi
+⏳️ Loading model from ml-model.mlem
+Starting fastapi server...
+💅 Adding route for /predict
+💅 Adding route for /predict_proba
+Checkout openapi docs at <http://0.0.0.0:8080/docs>
+INFO:     Started server process [5750]
+INFO:     Waiting for application startup.
+INFO:     Application startup complete.
+INFO:     Uvicorn running on http://0.0.0.0:8080 (Press CTRL+C to quit)
+```
+
+The second one allows you to package your models as a Python package, build a
+Docker Image, or export it as some other format (e.g. `.onnx` which is coming
+soon).
+
+```shell
+$ mlem pack mlem-model pip -c package_name=mlem-translate -c target=build/
+⏳️ Loading model from ml-model.mlem
+💼 Written `ml-package` package data to `build`
+$ tree build/
+build
+├── MANIFEST.in
+├── ml-package
+│   ├── __init__.py
+│   ├── model
+│   └── model.mlem
+├── requirements.txt
+└── setup.py
+```
+
+The last one allows you to deploy models to deployment providers, such as Heroku
+(AWS Sagemaker and Kubernetes are coming).
+
+```shell
+$ mlem deploy create myservice -m mlem-model -t staging -c app_name=mlem-quick-start
+⏳️ Loading deployment from my-service.mlem
+🔗 Loading link to staging.mlem
+🔗 Loading link to mlem-model.mlem
+💾 Updating deployment at my-service.mlem
+🛠 Creating docker image for heroku
+  🛠 Building MLEM wheel file...
+  💼 Adding model files...
+  🛠 Generating dockerfile...
+  💼 Adding sources...
+  💼 Generating requirements file...
+  🛠 Building docker image registry.heroku.com/mlem-quick-start/web...
+  ✅  Built docker image registry.heroku.com/mlem-quick-start/web
+  🔼 Pushing image registry.heroku.com/mlem-quick-start/web to registry.heroku.com
+  ✅  Pushed image registry.heroku.com/mlem-quick-start/web to registry.heroku.com
+💾 Updating deployment at my-service.mlem
+🛠 Releasing app mlem-quick-start formation
+💾 Updating deployment at my-service.mlem
+✅  Service mlem-quick-start is up. You can check it out at https://mlem-quick-start.herokuapp.com/
+```
+
+# Fast model registry setup based on Git
+
+[versions]: /doc/use-cases/versioning-data-and-model-files
+[mp]: /doc/start/metrics-parameters-plots
+[experiments]: /doc/user-guide/experiment-management
+[gto]: https://github.com/iterative/gto
+[mlem]: https://mlem.ai/
+[modeling process]: /doc/start/data-pipelines
+[remote storage]: /doc/command-reference/remote
+[sharing]: /doc/start/data-and-model-access
+[via cml]: https://cml.dev/doc/cml-with-dvc
+[gitops]: https://www.gitops.tech/
+
+MLEM is a core building block for a Git-based ML model registry, together with
+other Iterative tools, like GTO and DVC.
```

Comment:
GTO - other than at ODSC East and those that have found repo, we haven't really exposed GTO.  We probably need more links/explanation/docs/repo pointing here.

Reply:
ALso I'm realizing we need to address that in the image for Twitter....

Reply:
Yes, I think we need something. I'm also thinking about a technical page that explains how to set up MLEM + GTO + DVC together.